go.mod

```diff
@@ -3,3 +3,9 @@ module github.com/notaryproject/notation-core-go
 go 1.18
 
 require github.com/golang-jwt/jwt/v4 v4.4.2
+
+require (
+	github.com/fxamacker/cbor/v2 v2.4.0 // indirect
+	github.com/veraison/go-cose v1.0.0-rc.1
```

Comment:
Shouldn't we use `github.com/veraison/go-cose v1.0.0-rc.1.0.20220824135457-9d2fab636b83`?